### PR TITLE
troubleshooting openSSL

### PR DESCRIPTION
**Pull request by jburel**
add open ssl section to troubleshooting

cc @mtbc @joshmoore

### CODE REVIEW

File: omero/sysadmins/troubleshooting.txt [hunk 1]
```diff
@@ -493,3 +493,58 @@ Data corruption
 
 If you are dealing with a data corruption issue, you may find the information
 on :ref:`pixelresolutionorder` useful.
+
+Open SSL version
+^^^^^^^^^^^^^^^^
+
+With newer versions of OpenSSL i.e. version 1.0.2k or newer and 1.1.0 on Debian 9, weaker ciphers like ADH are disabled by default.
+This means that it is not possible to connect to an OMERO.server using any OMERO clients e.g. the Java Desktop client.
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: end with comma

File: omero/sysadmins/troubleshooting.txt [hunk 2]
```diff
@@ -493,3 +493,58 @@ Data corruption
...
+the OMERO.web client or the CLI.
+The parameter ``@SECLEVEL=0``, enabling the weaker ciphers, needs to be added in two files in order to allow connection.
+
+In ``etc/templates/grid/templates.xml``,
+
+replaces line ``108``
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: "replace"

File: omero/sysadmins/troubleshooting.txt [hunk 3]
```diff
@@ -493,3 +493,58 @@ Data corruption
...
+by
+
+::
+    
+    <property name="IceSSL.Ciphers" value="ADH:!LOW:!MD5:!EXP:!3DES:@STRENGTH:@SECLEVEL=0"/>
+
+
+In ``lib/python/omero/clients.py``,
+
+ depending on the Ice version used, replaces line ``207`` (Ice 3.6) or ``209`` (Ice 3.5)
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: "replace"

**Comment by manics** [comment]
How about add a sentence saying this fix will be included in the next release?

**Comment by jburel** [comment]
It won't be included in the coming 5.3.3 since it will break existing systems and more work will be required

**Comment by hflynn** [comment]
Renders fine on staging but there is a lot of white space in the file - you can put the `::` at the end of the previous line of text so you just need the blank lines before and after the code blocks. Also, do you have your editor set to a wider line wrapping than 78 characters?

**Comment by jburel** [comment]
@hflynn should be fixed now

**Comment by hflynn** [comment]
👍 thanks

**Comment by hflynn** [comment]
You happy for this to be merged now @mtbc ?

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/troubleshooting.html#open-ssl-version.

File: omero/sysadmins/troubleshooting.txt [hunk 4]
```diff
@@ -493,3 +493,48 @@ Data corruption
 
 If you are dealing with a data corruption issue, you may find the information
 on :ref:`pixelresolutionorder` useful.
+
+Open SSL version
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: "OpenSSL" no space

**Comment by mtbc** [comment]
The specific line numbers will be interesting to maintain: we will need to check for changes before each release.

**Comment by jburel** [comment]
The only option will be to copy across the template but at this stage I don't think we should do that
ideally we should determine the version installed and adjust the template accordingly
This will require more work

**Comment by jburel** [comment]
@mtbc done

**Review by mtbc — APPROVED** [review]

**Pull request merged**